crates/hyperqueue/src/client/commands/directives.rs

```diff
@@ -0,0 +1,146 @@
+use crate::client::commands::submit::SubmitJobConfOpts;
```

Comment:
I would put this file into the parent directory (`client`), or create a new directory `commands/submit` and put it there. The files in this directory should more or less correspond to individual command (or command groups).